src/core/shape/plane.js

```diff
@@ -109,7 +107,8 @@ class Plane {
      * @returns {this} A duplicate Plane.
      */
     clone() {
-        return new this.constructor(this.point, this.normal);
+        const clone = new this.constructor();
+        return clone.copy(this);
```

Comment:
```suggestion
        /** @type {this} */
        const cstr = this.constructor;
        return new cstr().copy(this);
```

Reply:
This seems to be the format for `clone` across the engine.

Reply:
hah, I copied from DepthState ;)